src/views/Home.jsx

```diff
@@ -34,14 +34,17 @@ export function Home({ setListToken }) {
 
 	return (
 		<div className="Home">
+			<h2>Welcome to your Smart Shopping List! </h2>{' '}
+			<h3>
+				You can create a new shopping list, or type in a token to view an
+				existing list.
+			</h3>
 			<button type="button" onClick={handleClick}>
-				Create new list
+				create new list
 			</button>
 			<p> - or - </p>
 			<form onSubmit={handleSumbit}>
-				<label htmlFor="tokenInput">
-					enter three word token to view existing list:
-				</label>
+				<label htmlFor="tokenInput">enter three word token:</label>
```

Comment:
When we're labeling controls (elements that the user interacts with), we're generally not writing commands to the user; we're writing _descriptions of the input we want._ In this specific case, we can delete the word "Enter" from this text, turning it from a command to a description.

Reply:
Oh that's so good to know EJ! Thanks for this tip! I'll update this as well :)